src/wavesurfer.js

```diff
@@ -441,6 +441,39 @@ var WaveSurfer = {
     },
 
     /**
+     * Unmounts the wavesurfer instance from the page, but keeps the audio
+     * player in memory
+     */
+    unmount: function (selector) {
+        if (this.container.children.length === 0) {
```

Comment:
This check is not correct. The container can have other elements besides wavesurfer.